spec/concepts/todo/create_spec.rb

```diff
@@ -1,7 +1,8 @@
 require 'rails_helper'
 
 RSpec.describe Todo::Create do
-  it "can be spec'd" do
-    raise "Implement me"
+  it "can create a todo" do
+    todo = Todo::Create.(todo: {title: "Testing helps!"}).model
+    expect(todo.title).to eq("Testing helps!")
```

Comment:
would be good also if you could test the invalid / failing ones / falsy ones... but this looks good and direct :+1: